graylog2-web-interface/src/components/extractors/extractors_configuration/GrokExtractorConfiguration.jsx

```diff
@@ -133,7 +133,7 @@ class GrokExtractorConfiguration extends React.Component {
         <Row>
           <Col mdOffset={2} md={1}>
             <Button bsStyle="info" onClick={this._onTryClick} disabled={this._isTryButtonDisabled()}>
-              {trying ? <i className="fa fa-spin fa-spinner" /> : 'Try against example'}
+              {trying ? <Icon name="spinner" spin /> : 'Try against example'}
```

Comment:
This and similar ones might also be replaceable with the `Spinner` component.

Reply:
Yea, i think this is a good idea, but will most likely do it within a follow-up PR